swdb_2019_tools/fs_utils.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+def fs_rsu_split(units_to_split,cutoff,return_hist=False):
+    ''' Accepts a pandas dataframe of units and splits units into fast-spiking and regular-spiking.
+        Useful for cell type analysis of neuropixels data.
+    Parameters
+    ==========
+    units_to_split : pandas.DataFrame
+        As returned by cache.get_units method
+    cutoff : float
+        Value of waveform duration to split cells on
+    return_hist : Boolean
+        If true, returns the histogram of waveform duration for units_to_split.
+        Useful for validating accurate cutoff.
+    
+    Returns
+    ==========
+    rsu_units : pandas.DataFrame
+        DataFrame of Regular Spiking Units (putative pyramidal cells and some interneurons)
+    fs_units : pandas.DataFrame
+        DataFrame of Fast Spiking Units (putative PV interneurons)
+    
+    Example
+    =======
+    >>> rsu_cells, fs_cells = fs_rsu_split(some_units,0.4,return_hist=True)
+    '''
+    
+    if ('duration' in units_to_split.columns) == True:
+        if return_hist == True:
+            plt.hist(units_to_split.duration,bins=60)
```

Comment:
i would make this ax that people can pass in, rather than plt, as you'll remove people's ability to control where that plot goes. Also .. return_hist isn't really an accurate name, plot_hist might be better.